src/Surfnet/StepupMiddleware/ManagementBundle/Tests/Controller/ConfigurationControllerTest.php

```diff
@@ -38,7 +38,7 @@ public function setUp()
         $this->loadFixtures([]);
 
         $this->client = static::createClient();
-        $this->password = $this->client->getKernel()->getContainer()->getParameter('management_password');
+        $this->password = 'management_password_test';
```

Comment:
the more I see this, the less I want this, please use the `parameters.yml` file. If needed, provide a fixture for test purposes instead. This reeks like magic strings that are related to something undiscoverable.